tests/parsing/ConfParserTest.py

```diff
@@ -200,3 +200,28 @@ def test_settings_override_warning(self):
                                             'already been defined in section '
                                             'name. The previous setting will '
                                             'be overridden.')
+
+
+class ConfParserTestParseSections(unittest.TestCase):
```

Comment:
Maybe I lost some context, but is a separate class necessary? Even if so, can you improve the name a bit

Reply:
I have modified the name to give more idea.🙂